web/stylelint.config.js

```diff
@@ -1,107 +1,11 @@
 /* eslint-disable no-magic-numbers */
 
 module.exports = {
-  processors: ['stylelint-processor-styled-components'],
-  plugins: ['stylelint-order'],
-  extends: ['stylelint-config-styled-components'],
-  syntax: 'scss',
+  extends: ['stylelint-config-recommended'],
+  customSyntax: 'postcss-jsx',
   rules: {
```

Comment:
@f1sh1918 any specific rules you'd like to keep? I figured it makes more sense to extend from a recommended rule config than to define all rules ourselves.

Reply:
it's fine for me to use recommended config. so far i found nothing to add